### DISCUSSION TITLE
guide/027/

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/scriptdev.ru#118] [Announcements]
# guide/027/

Справочник Typescript с примерами кода

https://scriptdev.ru/guide/027/

**Comment by Ruslan-web955** [comment]
Все примеры кода здесь и в других разделах устарели, с версии ts 5.3 больше невозможно получить доступ к полям родительского класса через super. См pr https://github.com/microsoft/TypeScript/pull/54056
